src/resources/index.js

```diff
@@ -135,9 +138,24 @@ module.exports = {
   },
   messages: {
     cli: {
+      installCompiler: (compiler, fromLocal) =>
+        `Installing ${compiler} from ${fromLocal ? 'local' : 'npm'}...`,
+      installCompilerSuccess: (template, compiler, version) =>
+        `${colors.green('✔')} Installed ${compiler} [${template.replace(
+          '-compiler',
+          ''
+        )} v${version}]`,
+      legacyTemplateDeprecationWarning: (legacyType, newType) =>
+        `Template-type "${legacyType}" has been deprecated and is now replaced by "${newType}"`,
+      startScaffold: () => `Scaffolding component...`,
+      scaffoldSuccess: componentPath =>
+        `${colors.green('✔')} Files created at ${componentPath}`,
+      creatingDir: () => `Creating directory...`,
+      createdDir: dirName =>
+        `${colors.green('✔')} Created directory "${dirName}"`,
```

Comment:
I am not sure I like the `${colors.green('✔')}` been here. The responsibility of this file should not be to have colors. I would rather create a new method in the logger, like `logger.okV('message')` which does this for you?